### PR TITLE
Add basic info about HTML entities

### PR DESCRIPTION
**Pull request by vzhikness**
**Description**

### CODE REVIEW

File: src/pages/html/html-entities/index.md [hunk 1]
```diff
@@ -3,13 +3,25 @@ title: HTML Entities
 ---
 ## HTML Entities
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/html/html-entities/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+In HTML characters like ```<```, ```>``` has special meaning. For example, if you use ```<``` character, browser will parse it as a beginning of an HTML element tag. Instead you need to use HTML entity&mdash;a string beginning with ```&``` and ending with ```;```. And for any special symbol there is an HTML entity.
```
Comment (jamesjpk123) [inline src/pages/html/html-entities/index.md]: Fix grammatical error "For example, if you use  ```<```  character, browser will parse" into "For example, if you use  **the** ```<```  character, **the** browser will parse"

Comment (jamesjpk123) [inline src/pages/html/html-entities/index.md]: Fix grammatical error "if you use ```<``` character," to "if you use **the** ```<``` character,"
Fix grammatical error "browser will parse it" to "**the** browser will parse it"
Fix grammatical error "you need to use HTML entity" to "you need to use an HTML entity"
Fix grammatical error "And for any special symbol there is" to "**(removal of 'and')** For any special symbol**,** there is"

File: src/pages/html/html-entities/index.md [hunk 2]
```diff
@@ -3,13 +3,25 @@ title: HTML Entities
...
+| &mdash;   | ```&mdash;``` |
+| &copy;    | ```&copy;```  |
+| &reg;     | ```&reg;```   |
+| &euro;    | ```&euro;```  |
+| &pound;   | ```&pound;``` |
+| &Theta;   | ```&Theta;``` |
+| &int;     | ```&int;```   |
 
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+Alternatively you can copy/paste UTF-8 (uncode) charaters directly to you HTML.
```
Comment (jamesjpk123) [inline src/pages/html/html-entities/index.md]: Fix grammatical error "characters directly to you HTML." to "characters directly into your HTML."

**Comment by vzhikness** [comment]
@jamesjpk123 thanks for your feedback! Please have a look.

File: src/pages/html/html-entities/index.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ title: HTML Entities
 ---
 ## HTML Entities
 
-In HTML characters like ```<```, ```>``` has special meaning. For example, if you use ```<``` character, browser will parse it as a beginning of an HTML element tag. Instead you need to use HTML entity&mdash;a string beginning with ```&``` and ending with ```;```. And for any special symbol there is an HTML entity.
+In HTML characters like ```<```, ```>``` has special meaning. For example, if you use the ```<``` character, your browser will parse it as a beginning of an HTML element tag. Instead you need to use an HTML entity&mdash;a string beginning with ```&``` and ending with ```;```. For any special symbol there is an HTML entity.
```
Comment (jamesjpk123) [inline src/pages/html/html-entities/index.md]: Grammatical fix "In HTML characters like" to "HTML characters like"
Grammatical fix "has special meaning" to "**have** special meaning"

File: src/pages/html/html-entities/index.md [hunk 4]
```diff
@@ -19,7 +19,7 @@ In HTML characters like ```<```, ```>``` has special meaning. For example, if yo
 | &Theta;   | ```&Theta;``` |
 | &int;     | ```&int;```   |
 
-Alternatively you can copy/paste UTF-8 (uncode) charaters directly to you HTML.
+Alternatively you can copy/paste UTF-8 (uncode) characters directly into your HTML.
```
Comment (jamesjpk123) [inline src/pages/html/html-entities/index.md]: Grammatical fix "(uncode)" to "(**unicode**)"

**Comment by vzhikness** [comment]
@jamesjpk123 Done! Thanks for such a detailed review.

**Comment by jamesjpk123** [comment]
@vzhikness this looks like an awesome article! It gets the OK from me!

**Comment by domwakeling** [comment]
Hey @vzhikness 

It seems that similar changes have already been accepted earlier for this article you're editing, sorry about that. 😓

If you feel you have more to add, please feel free to open up a new PR.

Thanks again! 😊

---

> If you have any questions, feel free to reach out by commenting below. 💬

**Pull request closed without merging**